IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/MSButton.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MSButton.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/20/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable public class MSButton: UIButton {
```

Comment:
1) Yes, you're right. I should improve that.

2) They're but sometimes things like the 'rounded' check doesn't update, but I guess it might be related to the first point you mentioned, I will check it out.